docs/reload-source-on-error.md

```diff
@@ -0,0 +1,43 @@
+# Using the reloadSourceOnError Plugin
+Call the plugin to activate it:
+
+```js
+player.reloadSourceOnError()
+```
+Now if the player encounters a fatal error during playback, it will automatically
+attempt to reload the current source. If the error was caused by a transient
+browser or networking problem, this can allow playback to continue with a minimum
+of disruption to your viewers.
+
+The plugin will only restart your player once in a 30 second time span so that your
+player doesn't get into a reload loop if it encounters non-transient errors. You
+can tweak the amount of time required between restarts by adjusting the
+`errorInterval` option.
+
+If your video URLs are time-sensitive, the original source could be invalid by the
+time an error occurs. If that's the case, you can provide a `getSource` callback
+to regenerate a valid source object. In your callback, the `this` keyword is a
+reference to the player that errored. The first argument to `getSource` is a
+function. Invoke that function and pass in your new source object when you're ready.
+
```

Comment:
```suggestion
### Example
```

Reply:
Uh, except the blank line should stay there

Reply:
Anyway this change isn't really necessary but it feels weird going right into it